bundles/AdminBundle/Controller/Admin/SettingsController.php

```diff
@@ -1506,25 +1506,17 @@ public function videoThumbnailUpdateAction(Request $request)
     /**
      * @Route("/robots-txt", methods={"GET"})
      *
-     * @param Request $request
-     *
      * @return JsonResponse
      */
-    public function robotsTxtGetAction(Request $request)
+    public function robotsTxtGetAction()
     {
         $this->checkPermission('robots.txt');
 
-        $robotsPath = $this->getRobotsTxtPath($request);
-
-        $data = '';
-        if (is_file($robotsPath)) {
-            $data = file_get_contents($robotsPath);
-        }
+        $config = Config::getRobotsConfig();
 
         return $this->adminJson([
             'success' => true,
-            'data' => $data,
-            'onFileSystem' => file_exists(PIMCORE_WEB_ROOT . '/robots.txt')
```

Comment:
Why you've removed this line? I think it's still a good thing to know if there's a robots.txt on the filesystem, that has priority over the one delivered by Pimcore. 
Could cause a lot of misunderstandings ...

Reply:
Thought nobody uses it... Will add it again